style.css

```diff
@@ -1,22 +1,25 @@
 
-main{
-    text-align: center;
-    margin:0%;
-    line-height: 1.8;
+main {
+  text-align: center;
+  margin: 0%;
+  line-height: 1.8;
 }
-img{
-max-width: 100%;
-height: auto;
-margin: auto;
-display: block; 
+
+img {
+  max-width: 100%;
+  height: auto;
+  margin: auto;
+  display: block;
 }
-#tribute-info{
-    margin:0 auto 30px ;
-    text-align: left;
-    line-height: 1.8;
-    max-width: 400px;
+
+#tribute-info {
+  margin: 0 auto 30px;
+  text-align: left;
+  line-height: 1.8;
+  max-width: 400px;
 }
-li{
-    margin: 20px 0;
-    font-size: medium;
```

Comment:
Align the heading with the unordered list for a better user experience.

Reply:
Hi @asharanjith . Good job on making this review. The language used however is not up to the code review language where you are required to not sound like you are giving orders. I suggest you use words like `kindly change this`, or `I suggest you make this change` 
<img width="1660" alt="Screenshot 2022-09-26 at 12 59 35" src="https://user-images.githubusercontent.com/48546992/192249258-49c8b1f4-7f79-4c70-8fde-b2a38e69ca3a.png">

Good job on explaining why the changes are required however and using the github review feature correctly